Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionMapsView.cs

```diff
@@ -12,6 +11,11 @@ namespace UnityEngine.InputSystem.Editor
     /// </summary>
     internal class ActionMapsView : ViewBase<ActionMapsView.ViewState>
     {
+        private const string kCmd_Rename = "Rename";
```

Comment:
I see commands are referenced via a string name, isn't there any central place where these are defined that's accessible from the package instead of redefining these? I suspect it would break if these changed or got superseded by something else?

Reply:
So, unfortunately there _is_ a central place... currently in IMGUI code 😄 Thread about this issue here:
https://unity.slack.com/archives/C3414V4UV/p1698856135642649

I messaged Antoine about it and he hasn't had time to move it out. Would making use of the IMGUI code be preferable here?

Reply:
I think it would be fine to use the IMGUI code, no?
Once they move to another place we could change them.
We currently still have some IMGUI dependencies anyway. And this specific one is not related to IMGUI functionality, just naming references.

Reply:
Sorry for delayed feedback here, must have overlooked you had an outstanding question to me here. I would suggest InputActionsEditorConstants.cs ?

Reply:
Fixed in 1bda88dac95a5a89f3ff692b85fb73db3f7fc59b.